<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestPhylogenyRdialog</class>
 <widget class="QDialog" name="QtTestPhylogenyRdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTestPhylogenyRdialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>660</width>
        <height>398</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_NewickToPhylogenyPng">
         <property name="text">
          <string>NewickToPhylogeny</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Full tree</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Extant species only</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="image_NewickToPhylogenyPng">
         <property name="text">
          <string>[PNG]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="image_NewickToPhylogenyExtant">
         <property name="text">
          <string>[PNG]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_NewickToLttPlot">
         <property name="text">
          <string>NewickToLttPlot</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="image_NewickToLttPlot">
         <property name="text">
          <string>[PNG]</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="image_NewickToLttPlotExtant">
         <property name="text">
          <string>[PNG]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
